crates/nu-cli/src/data/base.rs

```diff
@@ -99,14 +98,14 @@ impl CompareValues {
             CompareValues::String(left, right) => left.cmp(right),
             CompareValues::Date(left, right) => left.cmp(right),
             CompareValues::DateDuration(left, right) => {
-                use std::time::Duration;
-
-                // Create the datetime we're comparing against, as duration is an offset from now
-                let right: DateTime<Utc> = if *right < 0 {
-                    (SystemTime::now() + Duration::from_secs((*right * -1) as u64)).into()
-                } else {
-                    (SystemTime::now() - Duration::from_secs(*right as u64)).into()
-                };
+                // FIXME: Not sure how to handle the unwraps since this function should not fail.
+                // FIXME: Not sure if I could do something better with the Span.
+                let duration = Primitive::into_chrono_duration(
+                    Primitive::Duration(right.clone()),
+                    Span::unknown(),
+                )
+                .unwrap();
+                let right: DateTime<Utc> = Utc::now().checked_add_signed(duration).unwrap();
```

Comment:
For now you can use `.expect(...)` instead of `unwrap()`. This will get you past the clippy warnings.